65black/StackOverflow-lite
# #160381203 Upvote or Downvote an Answer

## What does this PR do?
Adds logic for the` PUT /api/v1/questions/:questionId/answers/:answerId/votes` endpoint.
The user must be signed in to access this endpoint.

If the user is signed in, then hitting this endpoint will either upvote or downvote an answer depending on the request body.

### Description of Task to be completed?
Have the following endpoints working:

> POST /api/v1/questions/:questionId/answers/:answerId/votes

### How should this be manually tested?
After cloning the repo, cd into it and run `npm run start-dev`. Follow these steps:

 - Open postman and hit the login endpoint first to get your auth token.
 - Hit the above endpoint making sure to replace the IDs with an existing question and answer.
- In your request body, add a key-value pair like so: `{vote: 'upvote'}` and hit send.
 - You should get a success response.

### Screenshots
![downvote](https://user-images.githubusercontent.com/36507972/45361556-6f4ae280-b5ca-11e8-9a47-a70e63c30c78.JPG)
![upvote](https://user-images.githubusercontent.com/36507972/45361557-6f4ae280-b5ca-11e8-9fa1-1669fae4e21c.JPG)

### What are the relevant pivotal tracker stories?
#160381203
